src/PostMeta.php

```diff
@@ -123,7 +123,7 @@ public function auth( $allowed, $meta_key, $object_id, $user_id, $cap, $caps ):
 	 * @return mixed Sanitized meta value.
 	 */
 	public function sanitize( $meta_value, $meta_key, $object_type ) { // phpcs:ignore VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
-		return $meta_value;
+		return sanitize_text_field( $meta_value );
```

Comment:
The objects should reflect WordPress's default configs, by default no sanitization is applied.

```suggestion
		return $meta_value;
```